### PR TITLE
add dynamic url

### PR DESCRIPTION
**Pull request by Dhruvi-Makwana**

### CODE REVIEW

File: Chatify/chat/static/js/chat.js [hunk 1]
```diff
@@ -65,7 +63,7 @@ app.controller('chatCtrl', function ($scope, $http) {
     })
 
     $scope.showChat = function (user) {
-        $scope.ps = new WebSocket(`ws://127.0.0.1:8000/ws/chat/message/${user.id}/`)
+        $scope.ps = new WebSocket(`ws://${url}/ws/chat/message/${user.id}/`)
```
Comment (yatishTT) [inline Chatify/chat/static/js/chat.js]: ```ws``` must be dynamic

File: Chatify/chat/static/js/base.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let url = window.location.host
+let get_full_url = window.location.href
+function getSocketScheme() { return window.location.protocol == "http:" ? "ws:" : ""; }
```
Comment (yatishTT) [inline Chatify/chat/static/js/base.js]: : "wss"

**Review by yatishTT — CHANGES_REQUESTED** [review]

File: Chatify/chat/api.py [hunk 3]
```diff
@@ -103,16 +103,6 @@ def get(self, request):
         return Response(status=status.HTTP_200_OK)
 
 
-class OnlineUsersAPI(APIView):
-    def get(self, request):
-        data = (
-            User.objects.filter(is_active=True)
-            .order_by("-id")
```
Comment (yatishTT) [inline Chatify/chat/api.py:110]: Why have you removed this?

**Review by yatishTT — CHANGES_REQUESTED** [review]

File: Chatify/chat/api.py (hunk 3 above)
Comment (Dhruvi-Makwana) [reply Chatify/chat/api.py:110]: in line no. 79 there is already a class.Here 2nd time class is added that's why i removed.

**Review by yatishTT — APPROVED** [review]

**Pull request merged**